Dollar/Dollar/Dollar.swift

```diff
@@ -893,6 +893,37 @@ public class $ {
         return result
     }
     
+    /// Return array elements grouped by a given key
+    ///
+    /// :param array The array to source from
+    /// :param key The key used to group elements
+    public class func groupBy<T, E: Equatable>(array: [[T: E]], key: T ) -> [[[T: E]]] {
```

Comment:
This API is not very helpful as it groups but you do not know what key value it is grouping by in the array easily.